dataland-frontend/tests/component/testUtils/ExistenceChecks.ts

```diff
@@ -2,9 +2,11 @@
  * Checks if an image is present
  * @param name the "alt" identifier of the image
  * @param fileName the file the image is expected to display
+ * @param parentElement the element whose content to search for the image
  */
-export function checkImage(name: string, fileName: string): void {
-  cy.get(`img[alt="${name}"]`)
+export function checkImage(name: string, fileName: string, parentElement: Cypress.Chainable = cy.get("body")): void {
```

Comment:
Maybe `alternativeText` is better than `name`, because it does not actually look for the `name` attribute of the image element.

Reply:
Will be pushed in the net 5 minutes.